src/mongo/db/curop.h

```diff
@@ -247,7 +247,8 @@ class CurOp {
         if (_dbprofile >= 2)
             return true;
 
-        return opMicros >= serverGlobalParams.slowMS * 1000LL;
+        return opMicros >= serverGlobalParams.slowMS * 1000LL
```

Comment:
Both of these checks are duplicated above. Please, make two functions:
`bool isSlow(opTime);`
`bool exceedCollScanLimit(docs);`